packages/irc/irc.py

```diff
@@ -11,27 +12,27 @@
 #  Receives messages from the Halibot base, relays them to the IRC server
 class IrcAgent(HalAgent):
 
-	options = {
-		'nickname': {
+	options = OrderedDict([
+		('nickname', {
 			'type'    : 'string',
 			'prompt'  : 'Nickname',
 			'default' : 'halibot',
-		},
-		'hostname': {
+		}),
+		('hostname', {
 			'type'    : 'string',
 			'prompt'  : 'Server hostname',
 			'default' : 'irc.freenode.net',
-		},
-		'port': {
+		}),
+		('port', {
 			'type'    : 'string',
 			'prompt'  : 'Server port',
 			'default' : '6667',
-		},
-		'channel': {
+		}),
+		('channel', {
 			'type'    : 'string',
 			'prompt'  : 'Channel to join',
-		},
-	}
+		}),
+	])
```

Comment:
I agree it is messy. The options could be put in a separate file, to perhaps separate the code from the mess. I am not aware of a better way of doing this right now though. A meeting topic perhaps.